pkgs/by-name/gh/ghostscan/package.nix

```diff
@@ -0,0 +1,37 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+  nix-update-script,
+}:
+
+rustPlatform.buildRustPackage {
+  pname = "ghostscan";
+  version = "0-unstable-2025-09-28";
```

Comment:
Let's see if we can get some tags -> https://github.com/h2337/ghostscan/issues/1